modules/core/shared/src/main/scala/io/circe/Error.scala

```diff
@@ -45,6 +45,11 @@ object ParsingFailure {
  * decoding history resulting in the failure.
  */
 sealed abstract class DecodingFailure(val reason: Reason) extends Error {
+  // Added to satisfy MiMA
+  def this(message: String) = {
```

Comment:
Does this deserve a deprecation message?

Reply:
Ah, I guess a constructor on a `sealed abstract class` isn't publicly usable anyway. 👍

Reply:
Oh lol, we've been through this already in https://github.com/circe/circe/pull/1906#discussion_r788915361.

Reply:
Thanks.  Been too long, i should have reviewed that previous PR.